Solutions/ESETPROTECT/Analytic Rules/ESETThreatDetected.yaml

```diff
@@ -0,0 +1,41 @@
+id: 64badfab-1dd8-4491-927b-3ca206fa9a17
+name: Threats detected by ESET
+description: |
+  'Escalates threats detected by ESET.'
+severity: Low
+requiredDataConnectors:
+  - connectorId: ESETPROTECT
+    dataTypes:
+      - Syslog
```

Comment:
If the detection query operates on Parser ,dataTypes should be the Parser name and not the table name. [Please refer this ](https://github.com/Azure/Azure-Sentinel/wiki/Query-Style-Guide#datatypes)for more details :